### PR TITLE
new term - style guide - added a preferred term

### PR DESCRIPTION
**Pull request by mmfred**
We had a request to use original in place of legacy.

### CODE REVIEW

**Comment by mmfred** [comment]
Hey folks - please don't merge. I don't see to be able to make this a draft, but can you treat it as a draft?

**Comment by paperclypse** [comment]
There you go, now it's a draft @mmfred :)

File: src/content/docs/style-guide/word-choice/usage-dictionary.mdx [hunk 1]
```diff
@@ -427,6 +427,13 @@ For consistency across New Relic content, here are our general word usage guidel
     
     Also avoid the `Thing: Tagline` format, as in `X-Ray sessions: Traces and thread profiles for key transactions`, unless having a title with keywords will help with SEO.
   </Collapser>
+  
+  <Collapser
+    id="legacy"
+    title="legacy"
+  >
+    Don't use. Instead, use [`original`](#original).
```
Comment (mmfred) [inline src/content/docs/style-guide/word-choice/usage-dictionary.mdx:435]:
```suggestion
    Don't use. Instead, use [original](#original).
```

**Comment by mmfred** [comment]
This is ready for review and merge if you're okay with it. Thanks!

**Review by bradleycamacho — DISMISSED** [review]

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**